### PR TITLE
Update alpine to 3.15.8

### PR DESCRIPTION
**Pull request by ScheererJ**
/kind enhancement

**What this PR does / why we need it**:
Update alpine to 3.15.8

**Which issue(s) this PR fixes**:
None.

**Special notes for your reviewer**:

### CODE REVIEW

File: config/images/images.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ images:
 - source: alpine
   destination: eu.gcr.io/gardener-project/3rd/alpine
   tags:
-  - 3.15.4
+  - 3.15.8
```
Comment (oliver-goetz) [inline config/images/images.yaml:24]: We should keep the old version too, that we can keep track on the images we already copied for previous versions.

Comment (timebertt) [reply config/images/images.yaml:24]: If that's the decision we should state this explicitly in `config/images/README.md`

Comment (ScheererJ) [reply config/images/images.yaml:25]: I kept the old version, but I am also a bit surprised that we want to keep the history in addition to version control (git) also in the file.

**Pull request merged**